### ISSUE TITLE
Temporarily Disable for an APP or 5 MINUTES

### ISSUE THREAD

**Issue by Oxtie** [AdguardTeam/AdguardForAndroid#3684]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [Y ] I am running the latest version
- [ Y] I checked the documentation and found no answer
- [Y ] I checked to make sure that this issue has not already been filed

### Problem Description

Many times I run into problematic complicated secure sites that malfunctions with Adguard (or requiring too many rule changes)

### Proposed Solution

Add option in Adguard (or Logger) to temporarily Disable filtering an APP (or Allow EVERYTHING for 5(xx) Minutes)

### Alternatives Considered

I've had enough of adding custom rules to make a website function normally (which works only after Adguard is disabled)

### Additional Information

This request could be applied to all platforms 

I tried to sign into a government website and got frustrated enough to write my request here

**Comment by TPS** [comment]
This seems legit enough to not let closed `stale`.

**Comment by Versty** [comment]
@Oxtie Closed as a duplicate of https://github.com/AdguardTeam/AdguardForAndroid/issues/4772.
